src/main/java/domain/crew/CrewGenerator.java

```diff
@@ -0,0 +1,42 @@
+package domain.crew;
+
+import domain.DisciplinaryStatus;
+import domain.dateTime.AttendanceDateTime;
+import domain.record.AttendanceRecord;
+import domain.record.AttendanceRecords;
+import domain.record.AttendanceStatusCounts;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeSet;
+import util.CsvReader;
+
+public class CrewGenerator {
+
+    public Crews generate(final String filePath) {
+        final List<String[]> crews = CsvReader.readFile(filePath);
+        final Map<Nickname, TreeSet<AttendanceRecord>> crewData = new HashMap<>();
+        for (final String[] crew : crews) {
+            final String inputNickname = crew[0];
+            final String inputDateTime = crew[1];
```

Comment:
```suggestion
            final String inputNickname = crew[NICKNAME_IDX];
            final String inputDateTime = crew[DATETIME_IDX];
```
매직 넘버를 상수 처리해주세요~ :)
다른 사람이 코드를 볼 때 좀 더 가독성 있도록 배려하는 포인트랍니다 ☺️